Diploma/PageObjects/HomePage.cs

```diff
@@ -0,0 +1,56 @@
+using Microsoft.Playwright;
+
+namespace Diploma.PageObjects;
+
+internal class HomePage(IPage? page)
+{
+    private const string HomePageUrl = "https://automationexercise.com/";
+
+    // Locators
+    private ILocator LoginButton => page!.Locator("//a[@href='/login']");
+    private ILocator EmailInput => page!.Locator("//input[@data-qa='login-email']");
+    private ILocator PasswordInput => page!.Locator("//input[@data-qa='login-password']");
+    private ILocator LoginSubmitButton => page!.Locator("//button[@data-qa='login-button']");
+
+    private ILocator UserLogin(string userLogin) => page!.GetByText($"Logged in as {userLogin}");
+    private ILocator DeleteAccountButton => page!.Locator("//a[@href='/delete_account']");
+    private ILocator DeleteAccountConfirmation => page!.Locator("//h2[@data-qa='account-deleted']");
+
+    // Methods
+    public async Task GoToHomePage()
```

Comment:
this can be named just Open(), so when it is used like _homePage.Open() it is clear what happens